tap_aggregator/src/server.rs

```diff
@@ -456,33 +464,36 @@ mod tests {
             .build(format!("http://127.0.0.1:{}", local_addr.port()))
             .unwrap();
 
-        // Create 100 receipts
+        // Create 300 receipts
         let mut receipts = Vec::new();
-        for _ in 1..100 {
+        for _ in 1..300 {
             receipts.push(
-                EIP712SignedMessage::new(Receipt::new(allocation_ids[0], 42).unwrap(), &keys.0)
-                    .await
-                    .unwrap(),
+                EIP712SignedMessage::new(
+                    Receipt::new(allocation_ids[0], u128::MAX / 1000).unwrap(),
+                    &keys.0,
+                )
+                .await
+                .unwrap(),
             );
         }
 
         // Skipping receipts validation in this test, aggregate_receipts assumes receipts are valid.
         // Create RAV through the JSON-RPC server.
-        // Test with only 10 receipts
+        // Test with only 250 receipts
         let res: Result<
             server::JsonRpcResponse<EIP712SignedMessage<ReceiptAggregateVoucher>>,
             jsonrpsee::core::Error,
         > = client
             .request(
                 "aggregate_receipts",
-                rpc_params!(api_version, &receipts[..10], None::<()>),
+                rpc_params!(api_version, &receipts[..250], None::<()>),
             )
             .await;
 
         assert!(res.is_ok());
 
         // Create RAV through the JSON-RPC server.
-        // Test with all 100 receipts
+        // Test with all 300 receipts
```

Comment:
```suggestion
        // Test with all receipts to exceed request size limit
```

Making this comment vague would allow it to not need to be updated each time this changes (or ending up being out of sync).